java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java

```diff
@@ -82,4 +88,25 @@ public void test_cluster_transactions() {
         Object[] results = clusterClient.exec(transaction, RANDOM).get();
         assertArrayEquals(expectedResult, results);
     }
+
+    @Test
+    @SneakyThrows
+    public void save() {
+        String error = "Background save already in progress";
+
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0")) {
+            Exception ex =
+                    assertThrows(
+                            ExecutionException.class,
+                            () -> clusterClient.exec(new ClusterTransaction().save()).get());
+            assertInstanceOf(RequestException.class, ex.getCause());
+            assertTrue(ex.getCause().getMessage().contains("Command not allowed inside a transaction"));
+        } else {
+            var transactionResponse =
+                    tryCommandWithExpectedError(
```

Comment:
can we try again if it randomly returns with an error that it's saving?

Reply:
yes, but it may still fail
We don't test save's logic, but we test command. That error says that command received by the server and handled